src/pages/en/guides/content-collections.mdx

```diff
@@ -121,6 +121,19 @@ If you **do not** already extend Astro's `strict` or `strictest` recommended Typ
 }
 ```
 
+:::note[Not using TypeScript?]
```

Comment:
I think this note looks good!

The only thing I'm wondering about is how explicitly to make sure people know that the section above (that's not in a note) ALSO (very likely) applies to peeps not using TypeScript.  someone's eyes might just glance here and see the ONE new line, when really, if they're not using TS, they won't have `strict` or `strictest`, so in fact, they'll need EVERYTHING in the code sample in the note.

This *could* be as simple as "If you're not using typescript, you'll ALSO have to..." (not sure whether in fact ALL the lines should be `+` diffs). Again, the worry is just that the "note" format is so visually strong, and says, "Not
 using TypeScript?" that I can totally see this pulling people into skipping the part about strictNullChecks above. So, this "note" might have to cover that, too.

(Which also means, should this in fact be a note, or just a "If you're not writing any TypeScript at all..." paragraph?)

Reply:
The code sample here does show the full `tsconfig` you need — `allowJs` and `strictNullChecks` — so maybe it’s enough to simply highlight _both_ of those as new/inserted lines?

Definitely think there’s an argument for paragraph/subhead over note though — happy either way.

Reply:
@sarah11918 Yeah I think making this note a paragraph will make the previous paragraph look less specific to TypeScript users, we won't even need to highlight both lines, we'll just add to the previous code block